samila/functions.py

```diff
@@ -339,6 +342,7 @@ def save_data_file(g, file_adr):
     try:
         with open(file_adr, 'w') as fp:
             json.dump(data, fp)
+        result["message"] = os.path.dirname(os.path.abspath(file_adr))
```

Comment:
I think a direct file address (`os.path.abspath(file_adr)`) is better than a folder address. isn't?